rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayDirectiveTree.scala

```diff
@@ -228,12 +229,11 @@ object DisplayDirectiveTree extends Loggable {
       )
 
       override def body = {
-
         val editButton = {
           if (addEditLink && ! directive.isSystem) {
             val tooltipId = Helpers.nextFuncName
             <span class="treeActions">
-              <span  class="tooltipable treeAction noRight directiveDetails fa fa-pencil" tooltipid={tooltipId} title="" onclick={redirectToDirectiveLink(directive.id).toJsCmd}> </span>
+              <span class="tooltipable treeAction noRight directiveDetails fa fa-pencil" tooltipid={tooltipId} title="" onclick={redirectToDirectiveLink(directive.id).toJsCmd}> </span>
```

Comment:
I think we could have updated that tooltip too (even if it only appears in the Rule page)